src/markup_bench.rs

```diff
@@ -0,0 +1,82 @@
+use criterion;
+
+// TODO: Switch to `markup::define!` when upgrading to Rust 2018.
```

Comment:
Best get that out of the way now? :)

Reply:
Sorry do you mean remove this comment or upgrade the crate to 2018?